### ISSUE TITLE
Jni type Display error

### ISSUE THREAD

**Issue by baikaishiuc** [Vector35/binaryninja-api#5090]
**Version and Platform (required):**
- Binary Ninja Version:[3.5.4526]
- OS: [Mac]
- OS Version: [14]
- CPU Architecture: [ M2]

**Bug Description:**
![image](https://github.com/Vector35/binaryninja-api/assets/618485/b257fcfd-940f-4694-bb0a-2db1ede5b018)

```
struct qh_env2 __packed
{
    struct qhJniEnv* qhJniEnv;
    JNIEnv* jniEnv;
};



```

**Steps To Reproduce:**

**Expected Behavior:**
It should be present as : (env2.qhJniEnv)->NewLocalRef

**Screenshots:**
If applicable, please add screenshots here to help explain your problem.

**Additional Information:**
Please add any other context about the problem here.

**Comment by plafosse** [comment]
Could you please share this binary?

**Comment by baikaishiuc** [comment]
> Could you please share this binary?

Yeah, I sent emai with bn to this address :   binaryninja@vector35.com

**Comment by galenbwill** [comment]
Vector 35 employees can find the binary by searching for `Brouhaha Gobbledygook Snickerdoodle Bumbershoot` in slack.

**Comment by plafosse** [comment]
I believe this is a duplicate of another issue but I can't find it.

**Comment by D0ntPanic** [comment]
Fixed in 4.1.5108
